saleor/order/tests/test_notifications.py

```diff
@@ -160,6 +160,46 @@ def test_get_order_line_payload(order_line):
     }
 
 
+def test_get_order_line_payload_deleted_variant(order_line):
+    order_line.variant = None
+    payload = get_order_line_payload(order_line)
+    unit_tax_amount = (
+        order_line.unit_price_gross_amount - order_line.unit_price_net_amount
+    )
+    total_gross = order_line.unit_price_gross * order_line.quantity
+    total_net = order_line.unit_price_net * order_line.quantity
+    total_tax = total_gross - total_net
+    assert payload == {
+        "variant": None,
```

Comment:
IMHO the standard payload is covered in different test. I would say that this is enough to only check if a variant field exists in the payload. (No need to twice check the same payload)

Reply:
It was also my concern. But I think other tests do not cover the payload with deleted variant so we can't be sure if any change made in code won't change anything in the whole payload in this specific case. What do you think?